File: server/src/com/twilightimperium/backend/model/RequestResponse/ActivateRequest.java
package com.twilightimperium.backend.model.RequestResponse;

import com.twilightimperium.backend.model.game.Location;

public class ActivateRequest {
    private final Location coords;

    public ActivateRequest(Location coords) {
        this.coords = coords;
    }

    public Location getCoords() {
        return this.coords;
    }


}

File: client/lib/model/request_response/activate/activate_request.dart
class ActivateRequest {
  ActivateRequest(int x, int y) {
    coords = Location(x, y);
  }
  late final Location coords;

  Map<String, dynamic> toJson() => {
    'coords': coords.toJson()
  };
}

class Location {
  final int x;
  final int y;

  Location(this.x, this.y);

  Map<String, dynamic> toJson() => {
    'x': x,
    'y': y
  };
}
